### PR TITLE
Add INAI Yucatan Spider

### LINKED ISSUE #691: INAIP Yucatán
**Links to bulk download and/or API documentation**
https://contratacionesabiertas.inaipyucatan.org.mx/contratacionesabiertas/datosabiertos#

**Link to CRM issue**
https://crm.open-contracting.org/issues/7082

**Priority**
e.g. anytime

### PR DESCRIPTION
**Pull request by cecicasco**
closes #691

### CODE REVIEW

File: kingfisher_scrapy/spiders/mexico_yucatan.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from kingfisher_scrapy.base_spider import PeriodicSpider
+from kingfisher_scrapy.util import components
+
+
+class MexicoYucatan(PeriodicSpider):
+    """
+       Domain
+         Instituto Nacional de Transparencia, Acceso a la Información y Protección de Datos Personales (INAI) - Yucatan
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_yucatan.py:8]:
```suggestion
        Instituto Estatal de Transparencia, Acceso a la Información Pública y Protección de Datos Personales
 (INAIP) - Yucatan
```

File: kingfisher_scrapy/spiders/mexico_yucatan.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        Download only releases from this date onward (YYYY format).
+        If ``from_date`` is not provided defaults to 2020.
+      until_date
+        Download only releases until this date (YYYY format).
+        If ``until_date`` is not provided defaults to 2020.
+       Bulk download documentation
+         https://contratacionesabiertas.inaipyucatan.org.mx/contratacionesabiertas/datosabiertos#
+    """
+
+    name = 'mexico_yucatan'
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_yucatan.py:20]:
```suggestion
    name = 'mexico_yucatan_inaip'
```

File: kingfisher_scrapy/spiders/mexico_yucatan.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from kingfisher_scrapy.base_spider import PeriodicSpider
+from kingfisher_scrapy.util import components
+
+
+class MexicoYucatan(PeriodicSpider):
+    """
+       Domain
+         Instituto Nacional de Transparencia, Acceso a la Información y Protección de Datos Personales (INAI) - Yucatan
+       Spider arguments
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_yucatan.py:9]: I think the indentation is incorrect here

**Review by yolile — CHANGES_REQUESTED** [review]
and please remember to run: https://kingfisher-collect.readthedocs.io/en/latest/contributing/index.html#commit-the-spider

**Comment by cecicasco** [comment]
> and please remember to run: https://kingfisher-collect.readthedocs.io/en/latest/contributing/index.html#commit-the-spider

Sorry.. I forgot!! I did it know..

File: kingfisher_scrapy/spiders/mexico_yucatan.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
+from kingfisher_scrapy.base_spider import PeriodicSpider
+from kingfisher_scrapy.util import components
+
+
+class MexicoYucatan(PeriodicSpider):
+    """
+    Domain
+        Instituto Nacional de Transparencia, Acceso a la Información y Protección de Datos Personales (INAI) - Yucatan
```
Comment (yolile) [inline kingfisher_scrapy/spiders/mexico_yucatan.py:8]:
```suggestion
        Instituto Estatal de Transparencia, Acceso a la Información Pública y Protección de Datos Personales
 (INAIP) - Yucatan
```

**Review by yolile — APPROVED** [review]

**Pull request merged**